src/components/Rockets.js

```diff
@@ -1,4 +1,44 @@
-const Rockets = () => (
-  <div>Rockets</div>
-);
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import {
+  selectRockets,
+  fetchRockets,
+  getStatus,
+  reserveRocket,
+} from '../redux/rockets/rocketsSlice';
+import styles from '../styles/Rockets.module.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector(selectRockets);
+  const status = useSelector(getStatus);
+
+  useEffect(() => {
+    if (status === false) dispatch(fetchRockets());
+  }, [dispatch, status]);
+
+  let rocketList;
+  if (status === true) {
+    rocketList = rockets.map((each) => (
```

Comment:
You can use the conciser `{status && ...}` syntax for conditional rendering which, I think, would be easier to read